### PR TITLE
fix(range-slider): fixes bound definition #463

### PR DESCRIPTION
**Pull request by bobylito**
in the case other component use the same attribute as conjunctive facet.

Also updates the version of noUISlider to fix part of the bounds
problems.

### CODE REVIEW

**Comment by bobylito** [comment]
Needs a PR to be merged on react-nouislider :) (hence the missing library)

**Comment by bobylito** [comment]
I added other related fixes...

**Comment by redox** [comment]
The price-ranges updates look good to me, not sure why I played with those `± 1` :+1:

**Comment by vvo** [comment]
Need to do PR on develop and maybe update the tests?

I still fail to get the big picture of this issue and understand why we need the changes in react-nouislider

File: widgets/range-slider/range-slider.js [hunk 1]
```diff
@@ -76,16 +76,23 @@ function rangeSlider({
...
+        helper.addNumericRefinement(attributeName, '>=', Math.round(newValues[0]));
       }
       if (newValues[1] < stats.max) {
-        helper.addNumericRefinement(attributeName, '<=', newValues[1]);
+        helper.addNumericRefinement(attributeName, '<=', Math.round(newValues[1]));
       }
       helper.search();
     },
+    getDisjunctiveFacetPosition(results, facetName) {
+      let facets = results.disjunctiveFacets;
```
Comment (vvo) [inline widgets/range-slider/range-slider.js]: Why not Array.find?

**Comment by bobylito** [comment]
I wil create a new PR (argh master) and do a screenshot so you can understand.

**Pull request closed without merging**